RCAIDE/Framework/Networks/Electric.py

```diff
@@ -222,41 +269,42 @@ def residuals(self,segment):
                 propulsor = bus.propulsors[list(bus.propulsors.keys())[0]]
                 propulsor.pack_propulsor_residuals(segment,bus)  
         return     
-    
+
     ## @ingroup Components-Energy-Networks
     def add_unknowns_and_residuals_to_segment(self, segment):
         """ This function sets up the information that the mission needs to run a mission segment using this network
-    
+
             Assumptions:
             None
-    
+
             Source:
             N/A
-    
+
             Inputs:
             segment
             estimated_battery_voltages                                    [v]
             estimated_rotor_power_coefficients                            [float]s
             estimated_battery_cell_temperature                            [Kelvin]
             estimated_battery_state_of_charges                            [unitless]
             estimated_battery_cell_currents                               [Amperes]
-            
+
             Outputs: 
             segment
-    
+
             Properties Used:
-            N/A
+            N/Abattery_tms in item
         """              
-        busses   = segment.analyses.energy.vehicle.networks.electric.busses 
+        busses          = segment.analyses.energy.vehicle.networks.electric.busses
```

Comment:
align equal signs (I will call this block text in the future)

Reply:
Done